spec/requests/users_controller_spec.rb

```diff
@@ -106,13 +151,20 @@
     describe 'PATCH #update' do
       context 'with valid parameters' do
         let(:new_attributes) do
-          { password: 'devise_sucks' }
+          attributes_for(:user, email: 'devise_sucks@gmail.com', password: 'devise_sucks')
+        end
+
+        it 'enqueues email to be delivered later' do
+          assert_enqueued_jobs 1 do
+            patch user_url, params: { user: valid_attributes }, headers: valid_headers, as: :json
+          end
         end
 
         it 'updates the requested user' do
           patch user_url, params: { user: new_attributes }, headers: valid_headers, as: :json
           user.reload
           expect(user.authenticate('devise_sucks')).to be_truthy
+          expect(user.unconfirmed_email).to eq('devise_sucks@gmail.com')
```

Comment:
Yeah I know this is my fault, but can we change these to use `new_attributes[:password` and `new_attributes[:email]` rather than hardcoding strings